### DISCUSSION TITLE
SSE using GET call

### DISCUSSION THREAD

**Discussion by sesmi123** [enisdenjo/graphql-sse#30] [Q&A] [answered]
Hi,

I am trying to use subscription over a graphql mesh service. I am able to achieve this from the graphiql mesh client. It uses a http SSE stream request over a GET request as shown below:
![image](https://user-images.githubusercontent.com/48611310/174619405-560e4161-a110-4ee0-9514-59608bf58151.png)

I need to subscribe to this from an angular app. I tried using the graphql-sse library but it is making the SSE request over a POST call. Is the request method configurable as GET/POST ?

Thankyou.

**Comment by enisdenjo** [comment, marked as answer]
If the server is declining requests from `graphql-sse`, then it does not implement the [GraphQL over Server-Sent Events Protocol](https://github.com/enisdenjo/graphql-sse/blob/master/PROTOCOL.md).

They are most likely using the simple SSE approach through EventSource as showcased [here](https://www.graphql-yoga.com/docs/features/subscriptions#handling-subscriptions-on-the-client).

**Reply by sesmi123** [reply]
Thankyou, SSE using EventSource worked.
